cluster_config.go

```diff
@@ -134,6 +135,11 @@ type Config struct {
 	// libp2p host peerstore addresses. This file is regularly saved.
 	PeerstoreFile string
 
+	// AuthorizationCacheSpan is the duration upto which a peer will be
+	// considered to be not trusting current peer, if an rpc request failed
+	// because of authorization error.
+	AuthorizationCacheSpan time.Duration
```

Comment:
This variable has a bad name. Can you understand what it actually does from the current name and explanation?

`UnauthorizedRequestCacheSpan` would be slightly better. With a description saying that certain rpc requests are not performed to things in this cache.